src/pint/residuals.py

```diff
@@ -1287,3 +1288,14 @@ def dof(self) -> int:
     def reduced_chi2(self) -> float:
         """Return the weighted reduced chi-squared."""
         return self.chi2 / self.dof
+
+    def calc_wideband_resids(self) -> np.ndarray:
```

Comment:
Again maybe have the docstring here point to the individual functions to make them easier to find.

Reply:
added.